### PR TITLE
VONK-2745: docu for lastN

### PR DESCRIPTION
**Pull request by eugene-blokhin**

### CODE REVIEW

File: features/lastn.rst [hunk 1]
```diff
@@ -0,0 +1,311 @@
...
+          ...
+          "Vonk.Plugin.LastN",
+        ],
+        "Exclude": [
+          ...
+        ]
+      }, ...etc...
+
+.. note::
+    We did not implement $lastn for all database types. Make sure the data database is configured for SQL Server.
```
Comment (cknaap) [inline features/lastn.rst:49]:
```suggestion
    We did not yet implement $lastn for all database types. Make sure the data database is configured for SQL Server.
```

File: features/lastn.rst [hunk 2]
```diff
@@ -0,0 +1,311 @@
...
+
+Examples
+--------
+
+The examples below use a predefined set of resources. You can add those resources to your instance of the server to reproduce the examples. To do it, please execute the following transaction bundle: :download:`pdf <../_static/files/lastN-example-bundle.json>`. The transaction bundle contains a patient with id = **8c5a23b8-5154-a665-8704-35f0e7a386e9** and a list of Observations for that patient.
```
Comment (cknaap) [inline features/lastn.rst:54]: "pdf"??

Comment (cknaap) [reply features/lastn.rst:54]: I'd give the resources in the bundle ids that a human can remember.

File: features/lastn.rst [hunk 3]
```diff
@@ -0,0 +1,311 @@
...
+Required parameters:
+
+* **patient** / **subject**: a reference to the Patient;
+* **category**: a category to get the most recent observations from.
+
+Optional parameters:
+
+* **max**: maximum number of Observations to return from each group;
+* other search parameters defined for the Observation resource.
+
```
Comment (cknaap) [inline features/lastn.rst:27]:
```suggestion
.. attention::
   Current limitations are:
   - $lastn has been implemented only for SQL Server databases.
   - the patient or subject reference must be a direct reference (using the ``id`` of the Patient), see the examples. A chained argument like ``patient.identifier=123`` is not yet supported.
```

**Review by cknaap — CHANGES_REQUESTED** [review]

**Review by alexzautke — APPROVED** [review]

**Pull request merged**